package.json

```diff
@@ -32,7 +32,7 @@
   "homepage": "https://github.com/resendlabs/resend-node#readme",
   "dependencies": {
     "@react-email/render": "0.0.7",
-    "type-fest": "3.13.0"
+    "node-fetch": "2.6.12"
```

Comment:
I think this was lingering from your merge conflicts, we removed node-fetch so this line can safely be removed 😄

Reply:
Fixed!